protocol-definitions/custom/Custom.yaml

```diff
@@ -174,15 +174,20 @@ customTypes:
         type: String
         nullable: false
         since: 2.0
-  - name: MapIndexConfig
+  - name: IndexConfig
     since: 2.0
+    returnWithFactory: true
     params:
-      - name: attribute
+      - name: name
         type: String
+        nullable: true
```

Comment:
is `name`  parameter nullable?

Reply:
Yes, the index name is optional. If not provided by user, it will be constructed automatically on the server-side.